api/internal/utils/pid.go

```diff
@@ -30,16 +30,14 @@ func WritePID(filepath string) error {
 		return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
 	}
 	pid := os.Getpid()
-	f, err := os.OpenFile(filepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC|os.O_CREATE, 0600)
+	f, err := os.OpenFile(filepath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
```

Comment:
Just want to know why we should change it.

Reply:
Hi, actually the `O_CREATE` flag had multiple occurrences :)